### PR TITLE
AntD Refactor: Remaining components of Setup

### PR DESCRIPTION
**Pull request by SanketDG**
Towards #531 

image

### CODE REVIEW

File: frontend/src/scenes/setup/UpdateEmailPreferences.js [hunk 1]
```diff
@@ -10,18 +11,20 @@ export function UpdateEmailPreferences() {
...
-                        userUpdateRequest({ user: { email_opt_in: !user.email_opt_in } })
-                        setSaved(true)
-                    }}
-                    defaultChecked={user.email_opt_in}
-                />
-                &nbsp;Tick this box to receive security and feature updates via email. You can easily unsubscribe at any
-                time.
+            <Switch
+                onChange={checked => {
+                    // TODO: refactor so we arent re rendering multiple times
```
Comment (timgl) [inline frontend/src/scenes/setup/UpdateEmailPreferences.js:16]: I tend not to be a super big fan of TODO comments, as they never tend to get done. Either we do it in the PR, raise an issue or just decide it's not important enough.

**Review by timgl — APPROVED** [review]
Looks great! Code reviewed and QA'd

File: frontend/src/scenes/setup/UpdateEmailPreferences.js (hunk 1 above)
Comment (SanketDG) [reply frontend/src/scenes/setup/UpdateEmailPreferences.js:16]: I agree. However, this was done in https://github.com/PostHog/posthog/blob/4afcde50b3ab477c83bff21d1f00ae9cd72e56f3/frontend/src/scenes/setup/UpdateEmailPreferences.js#L17

I have no idea what this means.

Do you have any insight on this or do you want me to ping the original author?

Comment (timgl) [reply frontend/src/scenes/setup/UpdateEmailPreferences.js:16]: Ah I missed that, feel free to just delete it

Comment (SanketDG) [reply frontend/src/scenes/setup/UpdateEmailPreferences.js]: Done!

**Comment by timgl** [comment]
Awesome, thanks!

**Pull request merged**